addon/models/document.js

```diff
@@ -14,7 +14,7 @@ export default class DocumentModel extends LocalizedModel {
   @hasMany files;
 
   get thumbnail() {
-    const thumbnail = this.files.filter((file) => file.type === "thumbnail")[0];
+    const thumbnail = this.files.findBy("type", "thumbnail");
```

Comment:
I would actually prefer a JS Array function since i think the ember array extension magic is a bit tricky as this is very confusing for new people where the heck these non JS Array functions come from (but im open for your opinion on this). (Or things like if you code non ember and are confused why the heck `firstObject` or `lastObject` is undefined :sweat: ) Would be fine by me if it were  `findBy(this.files, "type", "thumbnail")` but AFAIK `findBy` cant be imported separatly.
```suggestion
    const thumbnail = this.files.find(({type}) => type === "thumbnail");
```

Reply:
Doesn't the problem for new people hold true for every Ember function/method?

But this is just a suggestion. No pressure :smile: 

But I really hate destructuring in the parameters. So. I prefer to drop this PR.